lobe_android/app/src/main/java/com/example/test/tflite/ImageClassifier.java

```diff
@@ -26,28 +26,30 @@
 import org.tensorflow.lite.support.common.TensorOperator;
 import org.tensorflow.lite.support.common.ops.NormalizeOp;
 
-/** This TensorFlowLite classifier works with the float MobileNet model. */
-public class ClassifierFloatMobileNet extends Classifier {
+/** This TensorFlowLite classifier works with Lobe exported models. */
+public class ImageClassifier extends Classifier {
 
-  /** Float MobileNet requires additional normalization of the used input. */
+  /**
+   * Requires additional normalization of the used input.
+   */
   private static final float IMAGE_MEAN = 127.5f;
```

Comment:
we shouldn't need the mean/std values for images -- it should be a divide by 255.0 to get the pixels in the [0,1] range

Reply:
That makes sense. Since this "works" -- I will likely leave this as-is and then update as appropriate when using the input shapes from the model|signature.json